inlang/source-code/editor/src/pages/@host/@owner/@repository/+Page.tsx

```diff
@@ -86,6 +97,13 @@ function TheActualPage() {
 		})
 	)
 
+	const selectedLintRuleName = () => {
```

Comment:
It was suggested due to the "in-context" feel as you just selected the lint rule. This way the user gets the connection between the tip and the action.

**But** Jannes already asked, if this will inform the user only about this specific rule or about all lint issues.

@felixhaeberle Should I change it to "Get warned about new i18n issues in pull requests by adding the Ninja GitHub action"?

Reply:
Or "Automatically scan messages with lint rules in Pull Requests by adding the Ninja GitHub Action?"

Reply:
Can we settle with: "Get warned about lint reports in pull requests by adding the Ninja GitHub action."